product_multiple_barcodes/migrations/1.1/end-strategies.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2021 VentorTech OU
+# Part of Ventor modules. See LICENSE file for full copyright and licensing details.
+
+def migrate(cr, version):
+    cr.execute("DELETE FROM product_barcode_multi WHERE name IS NULL OR name = '' OR product_id IS NULL")
```

Comment:
Migration is not needed as it is new module and nobody installed it before. So folder can be removed

Reply:
OK